### PR TITLE
install daisyUI and test it

### LINKED ISSUE #5: configure a UI component library ( we are using tailwind so we need something that works with tailwind )
daisyUI is a good option

### PR DESCRIPTION
**Pull request by ayaali333**
close #5

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,11 +15,14 @@
     "autoprefixer": "10.4.14",
     "eslint": "8.44.0",
     "eslint-config-next": "13.4.7",
-    "next": "13.4.7",
+    "next": "^13.4.7",
     "postcss": "8.4.24",
-    "react": "18.2.0",
-    "react-dom": "18.2.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
```
Comment (MohammedAlabd) [inline package.json]: ❌  ❓ 
why did you change these?

File: src/pages/index.tsx [hunk 2]
```diff
@@ -1,118 +1,56 @@
-import Image from 'next/image'
-import { Inter } from 'next/font/google'
+import Image from "next/image";
+import { Inter } from "next/font/google";
```
Comment (MohammedAlabd) [inline src/pages/index.tsx]: ❌ 
you should put eslint/prettier changes in its own commit, or better in its own PR

**Review by MohammedAlabd — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (ayaali333) [reply package.json]: For some reason, when I run '**npm run dev**', I encounter the error message 'next' is not recognized as an internal or external command. Upon searching for a solution to this error, I found that the recommended solution is to run 'npm install react react-dom next'.

File: src/pages/index.tsx (hunk 2 above)
Comment (ayaali333) [reply src/pages/index.tsx]: I have no idea why those changed, I didn't do that

File: package.json (hunk 1 above)
Comment (MohammedAlabd) [reply package.json:21]: then you should not commit them,
if you want to do Es-lint fixes do them in a seperated commit

File: src/pages/index.tsx (hunk 2 above)
Comment (MohammedAlabd) [reply src/pages/index.tsx]: then you should not commit them,
if you want to do Es-lint fixes do them in a seperated commit

File: package.json (hunk 1 above)
Comment (MohammedAlabd) [reply package.json]: maybe you should have tried `npm install` only
that might work for you

Comment (MohammedAlabd) [reply package.json]: check out this to know what this character means
https://stackoverflow.com/a/22345808

**Comment by MohammedAlabd** [comment]
also your PR is pointed to the main, it should point to staging

**Comment by MohammedAlabd** [comment]
the commit message doesn't follow [commitizen docs](https://github.com/commitizen/cz-cli), can you please fix it?
